src/llm/__init__.py

```diff
@@ -94,11 +94,11 @@ async def process_user_intent(session_id, message):
     if len(relevant_contents) > 0:
         prompt_templating.append(
             SystemMessagePromptTemplate.from_template(
-                f"{PROMPT.get('suggested')}\n\n{suggested_content}"
+                f"{PROMPT.get('suggested')}. {suggested_content}"
             )
         )
 
-    prompt_templating.append(HumanMessagePromptTemplate.from_template("{user_message}"))
+    prompt_templating.append(HumanMessagePromptTemplate.from_template("Responda a seguinte pergunta:\n{user_message}"))
```

Comment:
if the prompt is in English?
we can't assume that all users are using the knowledge base in Portuguese

Reply:
@avelino that's true, my bad!